include/dynamatic/Dialect/Handshake/HandshakeOps.td

```diff
@@ -503,8 +503,16 @@ def SourceOp : Handshake_Op<"source", [Pure]> {
   }];
 
   let results = (outs ControlType:$result);
-  
-  let assemblyFormat = "attr-dict";
+
+  let builders = [
+    // We provide a builder that doesn't require the result type.
+    // The result type will be a ControlType without extra signals.
```

Comment:
I remembered the term `SimpleControl`.
How about these:

Unless specified, source operations should emit simple control tokens
Unless specified, source operations should emit SimpleControl tokens
Unless specified, the result of source operation should be SimpleControl

Reply:
Oh we should definitely use the phrase SimpleControl in the comment, that's better
I don't like mixing emit and SimpleControl, since for me one is for tokens and one is a channel.

What about 
Unless specified, output type is SimpleControl